mod/caching/set/abstract/answer_table_cached_count.rb

```diff
@@ -51,7 +51,7 @@ def return_arg val
 
 # needed for "found_by" wql searches that refer to search results
 # of these cards
-def wql_hash
+def wql_from_content
```

Comment:
there are lots of other places still overriding wql_hash rather than wql_content.  Do those not work?  I would think the caching wouldn't be an issue if you override wql_hash, no?

Reply:
There are two methods involved now that cache their result: `wql_hash` and `fetch_query`. That's why I introduced the `cache_query?` method. Overriding `wql_hash` means we have to duplicate the code that handles filter and sort options.
At the other places where we override wql_hash the result doesn't change (like here where a a search is involved)